server/src/main/java/org/elasticsearch/common/blobstore/OperationPurpose.java

```diff
@@ -16,6 +16,7 @@
  */
 public enum OperationPurpose {
     SNAPSHOT("Snapshot"),
+    REPOSITORY_ANALYSIS("RepositoryAnalysis"),
```

Comment:
Should we also introduce a purpose for health checking (may be this will be done as a separate future change)?

Reply:
Possibly, but this specific change is motivated by #101100.